libcontainer/cgroups/fs2/create.go

```diff
@@ -10,7 +11,7 @@ import (
 	"github.com/opencontainers/runc/libcontainer/configs"
 )
 
-// neededControllers returns the string to write to cgroup.subtree_control,
+// neededControllers returns the minimal string that should be write to cgroup.subtree_control,
```

Comment:
`string to write to` was OK

if you want `should be`, it should be `written`.

Reply:
if you're not using neededControllers, why keep it?

Reply:
Rootless needs it.
https://github.com/opencontainers/runc/blob/744bbe8128754ecb30a18465281ec3e02ee0766f/libcontainer/cgroups/fs2/fs2.go#L77-L85
How about move it to fs2.go and change the implementation way.

Reply:
Yes please.